raft/src/main/java/org/apache/kafka/raft/ReplicatedCounter.java

```diff
@@ -157,13 +160,18 @@ public synchronized void handleSnapshot(SnapshotReader<Integer> reader) {
     public synchronized void handleLeaderChange(LeaderAndEpoch newLeader) {
         if (newLeader.isLeader(nodeId)) {
             log.debug("Counter uncommitted value initialized to {} after claiming leadership in epoch {}",
-                committed, newLeader);
+                    committed, newLeader);
             uncommitted = committed;
             claimedEpoch = OptionalInt.of(newLeader.epoch());
         } else {
             log.debug("Counter uncommitted value reset after resigning leadership");
             uncommitted = -1;
             claimedEpoch = OptionalInt.empty();
         }
+        handleSnapshotCalls = 0;
+    }
+
+    public int getHandleSnapshotCalls() {
```

Comment:
The Kafka project tends not to prefix accessors with `get`:
```suggestion
    public int handleSnapshotCalls() {
```

Reply:
done